models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -19,4 +19,9 @@ def rename_workspace(self, selected_workspace, newName):
         return RenameWorkspace(selected_workspace,newName)
 
     def save_nexus(self, workspace, path):
-        SaveNexus(workspace,path)
\ No newline at end of file
+        SaveNexus(workspace,path)
+
+    @staticmethod
+    def get_workspace_handle(workspace_name):
```

Comment:
What's the reason that this is a static method when all other methods are instance methods?